vbfml/scripts/err_analysis

```diff
@@ -170,15 +196,18 @@ def predict(
 
     predictions = []
     image_pixels = []
+    labels = []
     for ibatch in tqdm(
         range(len(validation_sequences["image"])), desc="Making predictions"
     ):
-        features, _, _ = validation_sequences["image"][ibatch]
-        predictions.append(model.predict(features).argmax(axis=1))
+        features, labels_oneshot, _ = validation_sequences["image"][ibatch]
+        predictions.append(model.predict(features)[:, 1])
         image_pixels.append(features)
+        labels.append(labels_oneshot.argmax(axis=1))
```

Comment:
Sorry nitpicking but, could you rename `labels_oneshot` -> `labels_onehot`? (this represents one-hot encoding of the labels)

Reply:
I misunderstood the meaning of one-hot my bad. 
I changed it back